js/src/components/paid-ads/billing-card/billing-card.js

```diff
@@ -15,21 +16,33 @@ import fallbackBillingUrl from './fallbackBillingUrl';
 import { GOOGLE_ADS_BILLING_STATUS } from '.~/constants';
 import './billing-card.scss';
 
-const { APPROVED } = GOOGLE_ADS_BILLING_STATUS;
+const { APPROVED, PENDING } = GOOGLE_ADS_BILLING_STATUS;
 
 /**
  * Renders a success notice card or a setup card according to the billing status
  * of the connected Google Ads account.
  */
 export default function BillingCard() {
+	const [ showNotice, setShowNotice ] = useState( false );
 	const { billingStatus, hasFinishedResolution } =
 		useGoogleAdsAccountBillingStatus();
 
+	useEffect( () => {
+		// only show notice if we are in pending state before
+		if ( billingStatus.status === PENDING && ! showNotice ) {
+			setShowNotice( true );
+		}
+	}, [ billingStatus, showNotice ] );
```

Comment:
The billing card is not shown after refreshing the webpage at step 4 of the onboarding flow.

![image](https://github.com/user-attachments/assets/b03c5327-5b78-4b24-bf20-ac39f3513322)

Reply:
Removed the logic.